javascript/src/outputDisplayer.js

```diff
@@ -2,7 +2,7 @@ import { HCXIntegrator } from "./hcx_integrator.js";
 import { HcxOperations } from "./utils/hcx_operations.js";
 
 const config = {
-  participantCode: "testprovider1.swasthmock@swasth-hcx-staging",
+  participant_code: "testprovider1.swasthmock@swasth-hcx-staging",
```

Comment:
why the fields is not reverted here?

Reply:
The request body takes participant code in this "participant_code" patter to make an API call